make_site.go

```diff
@@ -10,7 +10,7 @@ import (
 	"text/template"
 
 	"github.com/PuerkitoBio/goquery"
-	"github.com/avelino/slugify"
+	"github.com/avelino/awesome-go/pkg/slug"
```

Comment:
this package `github.com/avelino/slugify` not only treat `/` by `""`, it has other special character treatments

Reply:
Sure, it does. That's why it is still used and just wrapped within `awesome-go/pkg/slug` to add the more GitHub style output. Would you prefer to move it over to the `avelino/slugify` package or what do you mean?